src/Avalonia.Visuals/Media/FontFamily.cs

```diff
@@ -51,6 +52,12 @@ public FontFamily(string name, Uri source, Uri baseUri = null) : this(name)
         /// </summary>
         public static FontFamily Default => new FontFamily(String.Empty);
 
+        /// <summary>
+        /// Represents all font families in the system
+        /// </summary>
+        public static IEnumerable<FontFamily> SystemFontFamilies =>
```

Comment:
This should be cached

Reply:
@Gillibald new system font can be added in application runtime. Should it be cached?

Reply:
Can you guarantee that the underlying platform is always updating its internal collection?

Reply:
Since this won't be used too often, it might be good to just mention this is an expensive call.